### DISCUSSION TITLE
1.5.0-beta6

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#389] [Обновления]
> [!WARNING]
> Это бета версия и она может работать нестабильно. Обо всех проблемах рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)

## Список изменений
- Два разных хранилища IndexedDB и LocalStorage были заменены на одно - GM_Storage
- Фикс отправки запросов к прокси-серверу после отключения перевода стрима (#381, #359)
- Фикс updateURL, downloadURL и namespace для минифицированных версий и бета версий (#385)
- Включена подгрузка бета локализаций в бета версиях
- Все языки, перевод на которые может быть недоступен, были помечены с помощью `❌`

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0-beta5...1.5.0-beta6

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0-beta6'>1.5.0-beta6</a>.</em>

**Comment by NullVerdict** [comment]
При первой инициализации новой БД опции не работают, тоесть они есть, но не работают.
Надо отдельный раз перетыкивать чтобы всё работало штатно.
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/87e3d131-fd5f-4d20-a647-e44819a8d3a1)
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/1a82f36c-7b74-4eb3-81c4-9125de8b1041)

**Reply by ilyhalight** [reply]
пофиксил

**Reply by NullVerdict** [reply]
Почему сбросилась база после обновления? Там и должно быть?

**Reply by ilyhalight** [reply]
Да, так и задумано. В 6-й бете все сохранялось в один объект "ext-data" и уже он сохранялся в GM_Storage, а в 7-й бете у каждой настройки, теперь, свое поле в GM_Storage, что позволяет легко задать им значение по умолчанию

**Reply by NullVerdict** [reply]
Выставление audioProxy на 1 для людей с Украинском языком в браузере было вырезано?

**Reply by ilyhalight** [reply]
Верну в следующей бете
